docs/src/conf.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+import subprocess
+from datetime import datetime
+
+ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", ".."))
+sys.path.insert(0, os.path.abspath("."))
+sys.path.insert(0, ROOT)
+
+
+# -- Project information -----------------------------------------------------
+
+project = "vesin"
+copyright = f"{datetime.now().date().year}, vesin developers"
+
+subprocess.run(["doxygen", "Doxyfile"], cwd=os.path.join(ROOT, "docs"))
```

Comment:
this should run inside a `setup(app)` function, so to only run it when sphinx actually run and not when the file is imported. See https://github.com/lab-cosmo/metatensor/blob/bf8c0130961b61d9a0dd10c676cf8570f611cc57/docs/src/conf.py#L100